apps/journeys-admin/src/components/TemplateView/TemplateTags/TemplateTags.tsx

```diff
@@ -81,7 +81,6 @@ function SwiperWrapper({ children }: SwiperWrapperProps): ReactElement {
         slidesPerView="auto"
         spaceBetween={8}
         breakpoints={swiperBreakpoints}
```

Comment:
```suggestion
        breakpoints={swiperBreakpoints}
```
      mousewheel={{
        forceToAxis: true
      }}